.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*               @jellyfin/web
```

Comment:
```suggestion
```

Reply:
No one has commented again so I'll commit this and merge to clean out the active pull requests for now. We can always revisit this in chat later.